content/maintainers-guide/qa/issues-management.md

```diff
@@ -0,0 +1,456 @@
+6. `**<Category>**` - **`BO`** (Back-office), **`FO`** (Front-office) or  **`WS`** (Webservices) ... - Which part of the software the feature bug/feature is applied
+7. `**<Page / Section>**` - e.g. **`Product`** Which page/section the feature bug/feature is applied
+8. **`Module`** and **`<Module name>`** – Name of the native module the bug / feature belongs to
+
+<aside>
+⚠️ Don’t use the **`develop`**  label, try to specify the branch like `**1.7.8.x**` or `**8.x**`
+
+</aside>
+
+#### Flow overview
+
+<img loading="lazy" class="image-with-shadow" src="/maintainers-guide/qa/images/Issue_state_flow.jpg">
+
+## Your routine
+
+#### New issues
+
+The first thing to do is to deal with the new issues. This can be done by [filtering out New issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+is%3Aopen+label%3ANew). Each of these issues must be managed according to the issue processing protocol.
+
+<aside>
+⚠️ This step should be repeated regularly during the day, to deal with issues as they arise.
+</aside>
+
+#### Issues with the NMI label
+
+The second step is to deal with issues labelled `NMI` (Need More Info) without `Waiting for author` label, which could have been updated by the author. To do this, you have to display the [NMI labeled issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aopen+is%3Aissue+label%3ANMI+-label%3A%22Waiting+for+author%22+sort%3Aupdated-asc) without `Waiting for author` and sort them by decreasing date of update. NMI issues are issues where crucial information is missing that only the author can provide, usually details on the reproduction steps. If the author has replied and provided the requested details, the issue will proceed according to the issue handling protocol.
+
+<aside>
+⚠️ Make sure you inspect all issues updated in the last few days and not just in the last 24 hours!
+</aside>
+
+#### Issues to be closed
+
+Finally, the third step consists in tracing the issues that have not been updated by the author for more than 20 days, and closing them. To do this, you have to [filter by increasing date](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aopen+is%3Aissue+label%3ANMI+label%3A%22Waiting+for+author%22+sort%3Aupdated-asc) and on the `NMI` + `Waiting for author`filter. Steps :
+
+1. Comment with the "Issue without answer" model
+2. Remove the `NMI` label
+3. Add the label `Invalid`
+4. Close the issue
+
+#### End of the day
+
+Take the time to list the issues reproduced and taken into account during the day. Send an email to [coreteam@prestashop.com](mailto:coreteam@prestashop.com), [support-team@prestashop.com](mailto:support-team@prestashop.com), [qa-core@prestashop.com](mailto:qa-core@prestashop.com) AND [coreproduct@prestashop.com](mailto:coreproduct@prestashop.com) with a brief report, with a list of validated issues (aka taken into account) and duplicated issues. Example:
```

Comment:
This might be problematic if we publish it as is. I suggest not to publish these addresses or we will start getting spammed.

Can we find another way? Is this part really needed?

Reply:
ping @hibatallahAouadni

Reply:
You're right @eternoendless we should delete this part, thanks!